rpc/examples/echo/playwright.config.ts

```diff
@@ -17,7 +17,7 @@ export default defineConfig({
   /* Opt out of parallel tests on CI. */
   workers: process.env.CI ? 1 : 3,
   /* Reporter to use. See https://playwright.dev/docs/test-reporters */
-  reporter: process.env.CI ? "dot" : "html",
+  reporter: [["html", { open: "never" }]],
```

Comment:
Weird, we definitely don't need to use this option in app. I wonder what's different here?

Reply:
On the plus side, I find auto-open annoying personally, so I don't hate this!